src/setup/helpers.ts

```diff
@@ -107,3 +108,41 @@ export function hashSignedObj(obj): string {
   }
   return crypto.hashObj(obj, true)
 }
+
+/**
+@param rawPayload: any - The original payload stripped of the signatures
+@param sigs: Sign[] - The signatures to verify
+@param allowedPubkeys: {[pubkey: string]: DevSecurityLevel} - The public keys that are allowed to sign the payload
+@param minSigRequired: number - The minimum number of signatures required
+@param requiredSecurityLevel: DevSecurityLevel - The minimum security level required to sign the payload
+@returns boolean - True if the payload is signed by the required number of authorized public keys with the required security level
+**/
+export function verifyMultiSigs(
+  rawPayload: {}, 
```

Comment:
In verifyMultiSigs, we do not check for duplicate signs in the payload, which could result in a logic bypass.

Reply:
This is valid and changes are coming